widgetssdk/src/main/java/com/glia/widgets/chat/adapter/UploadAttachmentAdapter.java

```diff
@@ -110,6 +111,10 @@ public void onBind(FileAttachment attachment, ItemCallback callback) {
                     extensionTypeText.setText(extension);
                 }
             }
+
+            removeItemButton.setContentDescription(context.getString(R.string.glia_chat_attachment_remove_item_content_description, diplayName));
```

Comment:
very long lines here should be wrapped to improve readability.

Reply:
fixed